contracts/SportMarkets/Parlay/ParlayMarketData.sol

```diff
@@ -126,94 +126,6 @@ contract ParlayMarketData is Initializable, ProxyOwned, ProxyPausable {
         }
     }
 
-    function getUserParlays(address _userAccount) external view returns (address[] memory userAllParlays) {
-        userAllParlays = new address[](userNumOfParlays[_userAccount]);
-        for (uint i = 0; i < userNumOfParlays[_userAccount]; i++) {
-            userAllParlays[i] = userParlays[_userAccount][i];
-        }
-    }
-
-    function getAllParlaysForGamePosition(address _sportMarket, uint _position) external view returns (address[] memory) {
-        return _getAllParlaysForGamePosition(_sportMarket, _position);
-    }
-
-    function getAllParlaysForGames(address[] memory _sportMarket)
```

Comment:
Some of these might be used by spoongly or other integrators

Reply:
I doubt it, but will ask.

Reply:
I use this function in the bot